initrd/bin/oem-factory-reset

```diff
@@ -772,11 +778,10 @@ generate_checksums() {
 
 	DEBUG "Detach-signing boot files under kexec.sig: ${param_files}"
 
-	if sha256sum $param_files 2>/dev/null | gpg \
+	if sha256sum $param_files 2>/dev/null | gpg --detach-sign \
```

Comment:
cosmetic; just so its easier to search codebase for gpg --detach-sign operations.